test/simple/test-module-version.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+var assert = require('assert');
```

Comment:
Tests in v0.12 overwhelmingly use var. let and const have bugs or at least not the right semantics.